src/main/java/g1501_1600/s1529_minimum_suffix_flips/Solution.java

```diff
@@ -0,0 +1,9 @@
+package g1501_1600.s1529_minimum_suffix_flips;
+
+// #Medium #String #Greedy
+
+public class Solution {
+    public String decode(String value) {
```

Comment:
Please add solution

Reply:
Solutions 1478 and 1529 are already existed